app/poros/api_obituary.rb

```diff
@@ -0,0 +1,15 @@
+class ApiObituary
+  attr_reader :headline,
+              :abstract,
+              :web_url,
+              :pub_date,
+              :word_count
+              
+  def initialize(info)
+    @headline = info[:data][:attributes][:headline]
+    @abstract = info[:data][:attributes][:abstract]
```

Comment:
I'm curious about the abstract as opposed to the lead_paragraph and snippet that we get back from NYT. The data contained in these different attributes were sometimes identical and other times they varied. Did you explore this any further?